doc/rst/language/evolution.rst

```diff
@@ -8,6 +8,183 @@ capture significant language changes that have the possibility of breaking
 existing user codes or code samples from old presentations or papers that
 predated the changes.
 
+version 1.15, April 2017
+------------------------
+
+Version 1.15 includes several language changes to improve array semantics.
+
+array lexical scoping
+*********************
+
+Continuing on the changes for 1.12 described below in
+:ref:`readme-evolution.lexical-scoping`, using arrays beyond
+their scope is now a user error:
+
+.. code-block:: chapel
+
+  proc badBegin() {
+    var A: [1..10000] int;
+    begin {
+      A += 1;
+    }
+    // Error: A destroyed here at function end, but the begin could still be using it!
+  }
+
+
+Similarly, using a slice after an array has been destroyed is an error:
+
+.. code-block:: chapel
+
+  proc badBeginSlice() {
+    var A: [1..10000] int;
+    var slice => A[1..1000];
```

Comment:
Good point. Which reminds me, we probably want to describe => deprecation in evolution.rst. See #5885.